src/Files.App/ServicesImplementation/FileTagsService.cs

```diff
@@ -0,0 +1,80 @@
+﻿using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Filesystem;
+using Files.Backend.AppModels;
+using Files.Backend.Services;
+using Files.Backend.Services.Settings;
+using Files.Backend.ViewModels.FileTags;
+using Files.Sdk.Storage;
+using Files.Sdk.Storage.Extensions;
+using Files.Sdk.Storage.LocatableStorage;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Files.App.ServicesImplementation
+{
+	/// <inheritdoc cref="IFileTagsService"/>
+	internal sealed class FileTagsService : IFileTagsService
+	{
+		private IStorageService StorageService { get; } = Ioc.Default.GetRequiredService<IStorageService>();
+
+		private IFileTagsSettingsService FileTagsSettingsService { get; } = Ioc.Default.GetRequiredService<IFileTagsSettingsService>();
```

Comment:
Can you make them readonly and lowercase

Reply:
(See #11008)

Reply:
I think it's good time to revise on codestyle guidelines we have on the website. It's been quite some time since they have been last updated